src/multivariate/mvnormal.jl

```diff
@@ -258,24 +258,25 @@ logdetcov(d::MvNormal) = logdet(d.Σ)
 
 ### Evaluation
 
-sqmahal(d::MvNormal, x::AbstractVector) = invquad(d.Σ, x .- d.μ)
+sqmahal(d::MvNormal, x::AbstractVector) = PDMats.invquad(d.Σ, x .- d.μ)
 
 sqmahal!(r::AbstractVector, d::MvNormal, x::AbstractMatrix) =
-    invquad!(r, d.Σ, x .- d.μ)
+    PDMats.invquad!(r, d.Σ, x .- d.μ)
 
 gradlogpdf(d::MvNormal, x::AbstractVector{<:Real}) = -(d.Σ \ (x .- d.μ))
 
 # Sampling (for GenericMvNormal)
 
-_rand!(rng::AbstractRNG, d::MvNormal, x::VecOrMat) =
-    add!(unwhiten!(d.Σ, randn!(rng, x)), d.μ)
+function _rand!(rng::AbstractRNG, d::MvNormal, x::VecOrMat)
```

Comment:
The old one-line form seemed OK.

Reply:
Ok, I considered awkward to have a one-liner spread on multiple lines, but not a drama, reverted in last commit

Reply:
I interpret "one-liner" as referring to a single expression (i.e. no `;`). :-)

Reply:
yes got it :)